sami2py/_core.py

```diff
@@ -45,7 +49,7 @@ def run_model(tag='model_run', lat=0, lon=0, alt=300, year=2018, day=1,
               no_scale=1, o2_scale=1, he_scale=1, n2_scale=1, n_scale=1,
               Tinf_scale=1, Tn_scale=1., euv_scale=1,
               wind_scale=1, hwm_model=14,
-              fejer=True, ExB_drifts=np.zeros((10, 2)), ve01=0., exb_scale=1,
+              fejer=True, ExB_drifts=None, ve01=0., exb_scale=1,
```

Comment:
Style note: why `ExB_drifts` and `exb_scale`? Also `Tinf_scale` and `Tn_scale`, though these are less annoying.

Reply:
Documented in #150